### DISCUSSION TITLE
Fix `get_swagger_ui_html` attribute `presets` or allow configurability?

### DISCUSSION THREAD

**Discussion by EssKayz** [fastapi/fastapi#10459] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Currently the following openapi configuration does not work at all:

    ext_app = FastAPI(
        title="Testing",
        description="Ducks are rubber",
        docs_url='/docs',
        openapi_url='/openapi.json',
        swagger_ui_parameters={
            'layout': "StandaloneLayout"
        }
    )
```

### Description

The problem seems to be that the included `SwaggerUIBundle.SwaggerUIStandalonePreset` does not inlcude the StandaloneLayout for whatever reason :
```
    html += """
    presets: [
        SwaggerUIBundle.presets.apis,
        SwaggerUIBundle.SwaggerUIStandalonePreset
        ],
    })"""
```
If this is changed to the following, you get the "StandaloneLayout" of swagger, with the navbar enabled.:
```
 <script src="https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui-standalone-preset.js" charset="UTF-8"> </script></script>

...
    html += """
    presets: [
        SwaggerUIBundle.presets.apis,
        SwaggerUIStandalonePreset
        ],
    })"""
 ```
[edit] added the relevant standalone-preset.js

Should we allow for configuring these presets using new input variables (swagger_ui_parameters won't work here), or just fix the default value? Is there anything else that could break with this change?

### Operating System

Linux

### Operating System Details

Running dockerized `python3.10-slim` docker container on various linux  distros and macOS across our development team

### FastAPI Version

0.100.0 -> 0.103.0

### Pydantic Version

2.0.3

### Python Version

3.10.12

**Comment by YuriiMotov** [comment]
Docs: https://fastapi.tiangolo.com/how-to/configure-swagger-ui/#javascript-only-settings
